src/fides/api/ctl/database/database.py

```diff
@@ -60,6 +83,154 @@ def create_db_if_not_exists(database_url: str) -> None:
         create_database(database_url)
 
 
+async def load_default_policies() -> None:
+    """
+    Checks whether DSR execution policies exist in the database, and
+    inserts them to target a default set of data categories if not.
+    """
+    FIDESOPS_AUTOGENERATED_CLIENT_KEY = "fides_autogenerated_client"
+    FIDESOPS_AUTOGENERATED_STORAGE_KEY = "fidesops_autogenerated_storage_destination"
+    STRING_REWRITE_STRATEGY_NAME = "string_rewrite"
+    db_session = sync_session()
+
+    client = ClientDetail.get_by(
+        db=db_session,
+        field="fides_key",
+        value=FIDESOPS_AUTOGENERATED_CLIENT_KEY,
+    )
+    if not client:
+        client, _ = ClientDetail.create_client_and_secret(
+            db=db_session,
+            client_id_byte_length=CONFIG.security.oauth_client_id_length_bytes,
+            client_secret_byte_length=CONFIG.security.oauth_client_secret_length_bytes,
+            fides_key="fides_autogenerated_client",
+        )
+
+    client_id = client.id
+    default_data_categories = [
+        DataCategory("user.biometric").value,
+        DataCategory("user.biometric_health").value,
+        DataCategory("user.browsing_history").value,
+        DataCategory("user.childrens").value,
+        DataCategory("user.contact").value,
+        DataCategory("user.date_of_birth").value,
+        DataCategory("user.demographic").value,
+        DataCategory("user.device").value,
+        DataCategory("user.gender").value,
+        DataCategory("user.genetic").value,
+        DataCategory("user.government_id").value,
+        DataCategory("user.health_and_medical").value,
+        DataCategory("user.job_title").value,
+        DataCategory("user.location").value,
+        DataCategory("user.media_consumption").value,
+        DataCategory("user.name").value,
+        DataCategory("user.non_specific_age").value,
+        DataCategory("user.observed").value,
+        DataCategory("user.organization").value,
+        DataCategory("user.political_opinion").value,
+        DataCategory("user.profiling").value,
+        DataCategory("user.race").value,
+        DataCategory("user.religious_belief").value,
+        DataCategory("user.search_history").value,
+        DataCategory("user.sensor").value,
+        DataCategory("user.sexual_orientation").value,
+        DataCategory("user.social").value,
+        DataCategory("user.telemetry").value,
+        DataCategory("user.unique_id").value,
+        DataCategory("user.user_sensor").value,
+        DataCategory("user.workplace").value,
+    ]
+    access_policy = Policy.create_or_update(
+        db=db_session,
+        data={
+            "name": "Default Access Policy",
+            "key": "default_access_policy",
```

Comment:
This deserves a constant like the other keys, don't you think?

Reply:
fixed